recipes/ginkgo/all/conanfile.py

```diff
@@ -33,28 +33,43 @@ class GinkgoConan(ConanFile):
         "fPIC": [True, False],
         "openmp": [True, False],
         "cuda": [True, False],
+        "half": [True, False],
     }
     default_options = {
         "shared": False,
         "fPIC": False,
         "openmp": False,
         "cuda": False,
+        "half": False,
```

Comment:
Upsteram seems to be setting this to `True` by default, except for Windows
https://github.com/ginkgo-project/ginkgo/blob/d7e1450b6ba9ee90dbaa839f4b4b5a5ad59e28cc/CMakeLists.txt#L46-L51

Would it not make sense to do the same in the recipe?  (for cases like this, we can leave the value in `default_options` undefined, and conditionally set a value in the `config_options()` method - as long as it's only conditional on settings)

Reply:
I think the conan package was intended to disable all optional features by default and only enable those which are explicitly requested by the user. To leave it consistent, I also disabled the half precision feature by default.